### ISSUE TITLE
dotnet_style_require_accessibility_modifiers missing from C# Code Style settings UI

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn-analyzers#5923]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/dotnet_style_require_accessibility_modif/1684055)._

---
"Text Editor -> C# -> Code Style -> General" is missing "dotnet_style_require_accessibility_modifiers" under "Modifier preferences", causing IDE0040 errors in the code file.

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0040

This is especially annoying when working on Unity projects because they do not support . EditorConfig files.

---
#### Feedback Bot on 3/6/2022, 06:25 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

#### Feedback Bot on 3/7/2022, 10:01 AM: 

<p>Thank you for sharing your feedback! Our teams prioritize action on product issues with broad customer impact. See details at: <a target='_blank' href="https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq">https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq</a>. In case you need answers to common questions or need assisted support, be sure to use <a target='_blank' href="https://visualstudio.microsoft.com/vs/support/">https://visualstudio.microsoft.com/vs/support/</a>. We’ll keep you posted on any updates to this feedback.</p>

#### Feedback Bot on 3/7/2022, 11:28 PM: 

<p>This issue is currently being investigated. Our team will get back to you if either more information is needed, a workaround is available, or the issue is resolved.</p>

---
### Original Solutions
(no solutions)
